src/ui/manager.cpp

```diff
@@ -196,12 +196,6 @@ Manager::Manager(const os::WindowRef& nativeWindow)
   , m_lockedWindow(nullptr)
   , m_mouseButton(kButtonNone)
 {
-  // The native window can be nullptr when running tests
-  if (nativeWindow) {
-    nativeWindow->setUserData(&m_display);
```

Comment:
I'd do this anyway at the very beginning and move only the `setDragTarget()` for later.